src/figuresClasses.ts

```diff
@@ -1,19 +1,90 @@
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
+
 export interface Figure {
+  shape: Shape,
+  color: Color,
 
+  getArea (): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: Shape = 'triangle';
 
-}
+  constructor(
+    public color: Color,
+    private sideA: number,
+    private sideB: number,
+    private sideC: number,
+  ) {
+    this.checkProperties();
+  }
+
+  getArea(): number {
+    const semiPer = (this.sideA + this.sideB + this.sideC) / 2;
+    const A = semiPer - this.sideA;
+    const B = semiPer - this.sideB;
+    const C = semiPer - this.sideC;
 
-export class Circle {
+    return Number(Math.sqrt(semiPer * A * B * C).toFixed(2));
```

Comment:
Probably it's not the way of rounding that tests expected 🙂

Reply:
Task expect that "returns the area of the figure rounded down to hundredths". 
I use method instead of Math.trunc(Math.sqrt(semiPer * A * B * C) * 100) / 100 because this one is not round
"returns the area of the figure to **second number after point**".

Reply:
Check first comment please

Reply:
**rounded down** crap!!!!!